target/ppc/int_helper.c

```diff
@@ -1645,23 +1645,23 @@ void helper_vslo(ppc_avr_t *r, ppc_avr_t *a, ppc_avr_t *b)
 
 #if defined(HOST_WORDS_BIGENDIAN)
 #define VINSERT(suffix, element)                                            \
-    void helper_vinsert##suffix(ppc_avr_t *r, ppc_avr_t *b, uint32_t index) \
+    void helper_VINSERT##suffix(ppc_avr_t *r, ppc_avr_t *b, uint32_t index) \
     {                                                                       \
         memmove(&r->u8[index], &b->u8[8 - sizeof(r->element[0])],           \
```

Comment:
This is not directly related to your change, but maybe a good opportunity to use assignments instead of memmove, similar to what was done for vextdu*?